content/en/news/releases/1.14.x/announcing-1.14/change-notes/index.md

```diff
@@ -0,0 +1,176 @@
+
+- **Added** new configuration options to `istio-iptables` and `istio-clean-iptables`
+  for including/excluding certain user groups from interception of the outgoing traffic
+  generated by them.
+
+This feature is intended primarily for use on VMs, where system administrators need
+to restrain interception of the outgoing traffic down to a few applications instead
+of intercepting all outgoing traffic.
+
+By default, as before, Istio Sidecar will intercept outgoing traffic from all processes,
+no matter what user groups they are running under.
+
+To change this behavior, system administrators can now use 2 new environment variables
+supported by `istio-iptables` and `istio-clean-iptables` - `ISTIO_OUTBOUND_OWNER_GROUPS`
+and `ISTIO_OUTBOUND_OWNER_GROUPS_EXCLUDE`.
+
+`ISTIO_OUTBOUND_OWNER_GROUPS` - is a comma separated list of groups whose outgoing traffic
+should be redirected to Envoy (sidecar).
+A group can be specified either by name or by a numeric GID.
+The wildcard character `*` can be used to configure redirection of traffic from all groups
+(default).
+
+`ISTIO_OUTBOUND_OWNER_GROUPS_EXCLUDE` - is a comma separated list of groups whose outgoing
+traffic should be excluded from redirection to Envoy (sidecar).
+A group can be specified either by name or by a numeric GID.
+Only applies when traffic from all groups (i.e. `*`) is being redirected to Envoy (sidecar).
+
+`ISTIO_OUTBOUND_OWNER_GROUPS` and `ISTIO_OUTBOUND_OWNER_GROUPS_EXCLUDE` are mutually
+exclusive, use only one of them.
+
+For example, `ISTIO_OUTBOUND_OWNER_GROUPS=101,java` instructs to intercept outgoing traffic only from
+  those processes that run under one of the user groups `101` (by `GID`) or `java` (by name).
+`ISTIO_OUTBOUND_OWNER_GROUPS_EXCLUDE=root,202` instructs to intercept outgoing traffic
+  from all processes except for those that under one of the user groups `202` (by `GID`)
+  or `root` (by name).
+  ([Issue #37057](https://github.com/istio/istio/issues/37057))
+
+- **Added** the ability to automatically set SNI when `DestinationRules`
+  do not specify it and `ENABLE_AUTO_SNI` is enabled.
+
+- **Added** the ability to set `credentialName` based secret configuration
+  at sidecars for egress TLS traffic when `WorkloadSelector` is specified in `DestinationRule`,
+  provided the sidecar has permission to list secrets in the namespace where it resides.
+
+- **Added** support for `WorkloadSelector` in `DestinationRule`.
+
+- **Added** warning messages for users attempting to use IP addresses as SNI values in `VirtualService.TLSRoute.Match.SniHosts`
+  ([Issue #33401](https://github.com/istio/istio/issues/33401))
+
+- **Added** support REPLACE for virtual host in envoy filter.
+
+- **Added** an API, `runtimeValues`, to [Proxy Config](/docs/reference/config/istio.mesh.v1alpha1/#ProxyConfig) to configure Envoy runtime configuration using.  ([Issue #37202](https://github.com/istio/istio/issues/37202))
```

Comment:
grammer. Using what? Maybe for configuring Envoy runtime.

Reply:
Fixed the grammar problem.